### PR TITLE
ServiceMonitor needs an actual service

### PR DESCRIPTION
**Pull request by greut**
:bug:

### CODE REVIEW

File: charts/workbench-operator/templates/metrics-service.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  labels:
+    control-plane: controller-manager
+  {{- include "workbench-operator.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.metricsService.type }}
+  selector:
+    control-plane: controller-manager
+  {{- include "workbench-operator.selectorLabels" . | nindent 4 }}
+  ports:
+	{{- .Values.metricsService.ports | toYaml | nindent 2 }}
```
Comment (iDmple) [inline charts/workbench-operator/templates/metrics-service.yaml:14]:
```suggestion
  {{- .Values.metricsService.ports | toYaml | nindent 2 }}
```

File: charts/workbench-operator/Chart.yaml [hunk 2]
```diff
@@ -13,9 +13,9 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.1
+version: 0.2.2
```
Comment (iDmple) [inline charts/workbench-operator/Chart.yaml:16]: Attention, same version as in https://github.com/CHORUS-TRE/workbench-operator/pull/32

Comment (greut) [reply charts/workbench-operator/Chart.yaml:16]: yeah, I'll create only one tag. It's a manual op.

File: charts/workbench-operator/templates/metrics-service.yaml (hunk 1 above)
Comment (greut) [reply charts/workbench-operator/templates/metrics-service.yaml:14]: it's generated code.

Comment (greut) [reply charts/workbench-operator/templates/metrics-service.yaml:14]: https://github.com/arttor/helmify/pull/153

**Comment by greut** [comment]
ping @iDmple

**Review by iDmple — APPROVED** [review]

**Pull request merged**